provider/data/datetime/fingerprints.csv

```diff
@@ -35617,225 +35617,225 @@ time_zone/generic_short@1, yrl, -> kgp
 time_zone/generic_short@1, zh-Hant, 176B, 84B, 1944b724241800b8
 time_zone/generic_short@1, zh-SG, -> ta-MY
 time_zone/locations@1, <lookup>, 1097B, 218 identifiers
-time_zone/locations@1, <total>, 1979623B, 1951879B, 204 unique payloads
-time_zone/locations@1, af, 8161B, 8025B, 1cb532d1ca53b263
-time_zone/locations@1, ak, 8179B, 8043B, 61a5ec381137eb44
+time_zone/locations@1, <total>, 1435127B, 1407337B, 204 unique payloads
```

Comment:
Praise: 1'979'623B to 1'435'127B is a big enough improvement that I'm happy landing this despite my other comments.